packages/cli/lib/LocalDevManager.js

```diff
@@ -179,6 +219,32 @@ class LocalDevManager {
     process.exit(EXIT_CODES.SUCCESS);
   }
 
+  getActiveAppInstallationData() {
+    return fetchAppInstallationData(
+      this.targetAccountId,
+      this.projectId,
+      this.activeApp.config.uid,
```

Comment:
These variables are kind of a nightmare to read, but I don't think destructuring would help the code's readability overall. Not a blocker, but something to think about.

Reply:
Honestly, maybe it's more a reflection on how our BE is structured...

Reply:
Yeah unfortunately the endpoint just requires a LOT of info to work. Not sure if there's any winning for us here